test/DynamoCoreTests/Configuration/PreferenceSettingsTests.cs

```diff
@@ -320,6 +320,8 @@ public void TestImportCopySettings()
             var newSettings = PreferenceSettings.Load(newSettingslFilePath);
 
             // validation
+            Assert.IsTrue(newSettings.isValidInstance(), "The new settings file is invalid");
```

Comment:
it would be good to have a negative test as well.

Reply:
Done.